apache-jsp/src/main/config/modules/apache-jsp.mod

```diff
@@ -1,4 +1,4 @@
-# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+# DO NOT EDIT THIS FILE - Use start modules correctly - See: https://eclipse.dev/jetty/documentation/
```

Comment:
I think the "Use start modules correctly" is a bit scolding. Let's keep it factual:

```suggestion
# DO NOT EDIT THIS FILE - See: https://eclipse.dev/jetty/documentation/
```

Reply:
We removed the reference to `/current/startup-modules.html` as that is no longer a valid URL.

Pointing to the documentation alone is insufficient. (what do they look for?)
Perhaps it should read ...

```
# DO NOT EDIT THIS FILE - See startup modules documentation at https://eclipse.dev/jetty/documentation/
```

Reply:
Or ...

```
# DO NOT EDIT THIS FILE - See Jetty Start Mechanism in Operations Guide at https://eclipse.dev/jetty/documentation/
```

Reply:
Or ...

```
# DO NOT EDIT THIS FILE
# See https://eclipse.dev/jetty/documentation/ > Operations Guide > Jetty Start Mechanism
```

Reply:
'/current' under the documentation is really old and not a maintained documentation URL atm (last updated 3 yrs ago)

[Look here for current exposed layout.](https://github.com/eclipse/jetty-website/tree/master/documentation)

That is how things are currently structured, talking with @sbordet and @lorban yesterday we agreed we should remove the content under '/current' and just provide a redirect to '/documentation'